client/pokemon/pokemon.ctrl.js

```diff
@@ -33,7 +33,12 @@ module.exports = class Pokemon {
   }
   parseBoxViewProps () {
     this.parsedOt = replace3dsUnicodeChars(this.data.ot);
-    const tidSize = this.data.gen === 7 ? 6 : 5;
+
+    this.isKB = this.data.otGameId >= 24 && this.data.otGameId <= 29;
+    this.isAB = this.data.otGameId >= 30 && this.data.otGameId <= 33;
+    this.isVC = this.data.otGameId >= 35 && this.data.otGameId <= 38;
+
+    const tidSize = this.isAB ? 6 : 5;
```

Comment:
By having this use isAB which is derived from otGameId rather than gen, this also ends up fixing another existing bug that prevented the tidSize from being correctly determined in card view. Nice! 👍